help_text/fa13d9f7d625dfa5758b294ad1d308da.md

```diff
@@ -0,0 +1,24 @@
+This error means that there is an image (`img`) element on the page without an `alt` attribute.  An `alt` attribute should contain alternative for the image in cases where the image can not be presented.  Some examples of these cases are when the image does not load or when the page is read with a screen reader for the visually impaired.
```

Comment:
fixed in https://github.com/mfairchild365/metric_w3c_html/commit/4da5d59c1c036975f5209f4c63f80a14afe9950e